lessonbuilder/tool/src/webapp/css/Simplepagetool.css

```diff
@@ -522,7 +522,7 @@ a.edit-link:link, a.edit-link:visited, a.section-merge-link:link, a.section-merg
    border-color: #840;
    color: #4b2600;
 }
-.colyellow .commentDiv, .colyellow .studentContentTable, .colyellow .questionDiv .contentCol, .colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv, .colyellow .announcementsHeaderDiv .timeline-Header{
+.colyellow .commentDiv, .colyellow .studentContentTable, .colyellow .questionDiv .contentCol, .colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv, .colyellow .announcementsHeaderDiv, .timeline-Header{
```

Comment:
I think this is also wrong:
`.colyellow .checklistDiv. colyellow .forumSummaryHeaderDiv` 
while we're here shall we just fix this as well.